ui/packages/catalog-ui-search/src/main/webapp/component/search-form-list/search-form-list.view.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+/*global require*/
+
+const CustomElements = require('../../js/CustomElements.js')
+const Marionette = require('marionette')
+const Router = require('../router/router.js')
+const user = require('../singletons/user-instance')
+const SearchForm = require('../search-form/search-form')
+import * as React from 'react'
```

Comment:
Why `import * as` instead of just `import React from 'react'`?

Reply:
Good question. According to @djblue, `import * as React` is needed in Typescript files or else you get a bunch of type errors. Either work in this case though so I'll go with the simpler `import React`.